### ISSUE TITLE
Hmm.. not as good as SCCM Client Center v2.0

### ISSUE THREAD

**Issue by nazidk** [rzander/sccmclictr#76]
Your original tool was brilliant and just worked straight off the bat - this new one, just won't connect to anything. Windows 7 or 10 even the localhost won't connect. I suppost MS has native tools now in SCCM CB that can do similar things?

**Comment by mhomoky** [comment]
Original tool used WMI to communicate with CCM agent over DCOM (if I'm not mistaken).

This version, using WinRM, has fewer network requirements in that regard, hence in a larger enterprise may be easier to work with as you don't have to get other teams involved (WinRM can run on ports 80/443 as well as 5985/5986).

I agree however it may at first glance appear more difficult to get a connection.  Have you tried running elevated to connect to localhost?

I've no idea if you're familiar with Windows Remoting but you should ensure it's functional on your machine or your client before running this. Not too difficult with e.g. psexec and an elevated remote command prompt to fire it up. A quick one-line powershell command will do it.

If PS Remoting is completely new to you, Google for Don Jones' free guide: "Secrets of Powershell Remoting".

One of the main features I miss from the original is the current actions panel showing what's happening right now, without resorting to the client's logs.  And yes MS has produced some new tools but none are quite as good as this. Thanks Roger.

**Comment by bryan5989** [comment]
@nazidk We have an Win 7 Win 10 mixed environment and have no issues. We also have up to date .NET Frameworks on all systems (your admin workstation needs this), and winRM enabled on all systems. We used GP to achieve this (as well as managing the trusted hosts list). and yes, you must elevate in order to connect to localhost.

I believe the _only_ prerequisite for this app to work to connect remotely is win-rm access. (provided your workstation has .net,(but who doesn't at this point))

To test if winRM is working in your environment, just open an elevated powershell and attempt to 
```ps
Enter-PSSession -ComputerName $Computer
```
If all goes well, then WinRM is not your problem.

**Comment by SCCMAdmin** [comment]
@nazidk I agree the old 2.0.4.0 tool was great but it's outdated now and can't do most things you need to do. I do still use it repair WMI and if anything was repaired in the first page worth of the log, I continue to delete root\ccm namespace. It's invaluable for that!  However I disagree CCCM 1.0.4.x doesn't work. I too use it in an environment of 15,000+ systems for all Windows OSes.

@bryan5989 we didn't use GPO to configure WinRM, it was a compliance baseline. Where is there easy information to read on the GPO configuration steps? Thanks.
